admin-dev/themes/new-theme/js/app/pages/stock/mixins/product-desc.js

```diff
@@ -39,7 +39,7 @@ export default {
       const arr = this.product.combination_name.split(',');
       let attr = '';
       arr.forEach((attribute) => {
-        const value = attribute.split('-');
+        const value = attribute.split(' - ');
```

Comment:
Because it depends on the `getAttributesResume` (if I'm right), why not getting this value from the BO?
Same for line 39, I think it can have the same problem :p

Reply:
I improved.